clintfidel/WEconnect
# #157088066 User Should Be Able To Get Right Search Result

#### What does this PR do?
This PR fixes search in view users-business component

#### Description of Task to be completed?
This PR ensures search return right result in the view-business component
#### How should this be manually tested?
- visit application url [here](weconnectapp.herokuapp.com)
- Navigate to the signup page and create an account or Login if you already have an account
- Register a business then,
- You would be redirected to your business listing page
- search on businesses that you already posted, you should see a list of businesses based on your search

#### What are the relevant pivotal tracker stories?
[157088066](https://www.pivotaltracker.com/story/show/157088066)

#### Screen Shot
![image](https://user-images.githubusercontent.com/26988297/39256692-75162dcc-48a7-11e8-869f-eb91d121c5e1.png)


#### Any background context you want to add?
N/A
